cmd/snap/cmd_run_test.go

```diff
@@ -2064,3 +2067,72 @@ func (s *RunSuite) TestRunDebugLog(c *check.C) {
 	// and we've let the user know that logging was enabled
 	c.Check(logBuf.String(), testutil.Contains, "DEBUG: enabled debug logging of early snap startup")
 }
+
+func (s *RunSuite) TestDesktopIntegrationNoDBus(c *check.C) {
```

Comment:
as other tests in this file these need to use MockLogger otherwise the test run output is polluted by the logs

Reply:
Done.